skimage/morphology/gray.py

```diff
@@ -175,20 +131,35 @@ def erosion(image, footprint=None, out=None, shift_x=False, shift_y=False):
     out : ndarrays, optional
         The array to store the result of the morphology. If None is
         passed, a new array will be allocated.
-    shift_x, shift_y : bool, optional
-        shift footprint about center point. This only affects
-        eccentric footprints (i.e. footprint with even numbered
-        sides).
```

Comment:
Sorry, it's been a while and I may have forgotten previous discussions. According to the PR descriptions, shouldn't `erosion` and other function have a new `mirror` parameter to replace this one?

Reply:
Somewhere we decided it was better to let the user call `mirror_footprint` themselves. It keeps things simpler.